bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -702,6 +704,48 @@ int runCmd(CommandLine cmdLine) throws Exception {
         }
     }
 
+    /**
+     * Command to delete a given ledger.
+     */
+    class MarkLedgerReplicatedCmd extends MyCommand {
+
+        MarkLedgerReplicatedCmd() {
+            super(CMD_MARKLEDGERREPLICATED);
+            opts.addOption("l", "ledgerid", true, "Ledger ID");
```

Comment:
It's better to provide a list or range option to allow given a group of ledger id to mark. For example, we can use `,` to give a list of ledgers, "1,2,3,4,5,6". We also can use a range to specify the ledger, "1...10". 
This can avoid calling this command multiple times when there has many ledgers that need to be marked.

Reply:
Make sense.

Reply:
+1